src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -394,33 +386,33 @@ def _construct_job_status(self, job: Job, state: dict) -> dict:
             "listener_count": self._running_jobs[job.job_id]["refresh"],
         }
 
-    def _child_job_states(self, sub_job_list, app_id, app_tag):
+    def _child_job_states(self, child_job_list, app_id, app_tag):
```

Comment:
changing this over to using `child_job_list` instead of `sub_job_list`, since we don't use the term `sub_job`